universal-application-tool-0.0.1/app/views/BaseHtmlView.java

```diff
@@ -42,6 +42,14 @@ public Tag renderHeader(String headerText) {
     return ImmutableList.of(labelTag, br(), inputTag, br(), br());
   }
 
+  public ImmutableList<DomContent> textInputWithLabel(
```

Comment:
(blocking): what's the difference between these new methods and `textField` methods? Should we get rid of the `textField` methods?

Reply:
They're similar, and in the end we'll keep one or the other, but I don't think we'll have a clear picture of what methods go in here until we start getting rolling with Tailwind.

Reply:
can you group these together with the other ones?